python/OnnxUdf/example.py

```diff
@@ -0,0 +1,35 @@
+import openeo
+
+# This will be required to import onnxruntime in the UDF.
+dependencies_url = "https://artifactory.vgt.vito.be:443/auxdata-public/openeo/onnx_dependencies.zip"
+# You can upload your own model and paste the url here.
+model_url = "https://artifactory.vgt.vito.be:443/auxdata-public/openeo/test_onnx_model.zip"
+
+spatial_extent = dict(zip(["west", "south", "east", "north"], [8.908, 53.791, 8.96, 54.016]))
```

Comment:
This dict-zip-lists construct seems a bit convoluted, a normal dict construct will be shorter too
```suggestion
spatial_extent = {"west": 8.908, "south": 53.791, "east": 8.96, "north": 54.016}
```